packages/devtools_app/lib/src/screens/performance/performance_controller.dart

```diff
@@ -949,8 +989,9 @@ class PerformanceController extends DisposableController
     firstWellFormedFrameMicros = null;
     _selectedTimelineEventNotifier.value = null;
     _selectedFrameNotifier.value = null;
-    _processing.value = false;
+    _workTracker.clear();
     serviceManager.errorBadgeManager.clearErrors(PerformanceScreen.id);
+    data?.rebuildCountModel.clearAllCounts();
```

Comment:
this is redundant since we already call `data?.clear()` above, so we can remove this

Reply:
done